### PR TITLE
Adding further UI improvements

### PR DESCRIPTION
**Pull request by dewmini**

### CODE REVIEW

File: modules/extensions/process/publisher/process/apis/upload_bpmn.jag [hunk 1]
```diff
@@ -29,15 +29,21 @@ var bpmnVersion = String(request.getParameter("bpmnProcessVersion"));
...
+var responseObj = {};
 
 try {
     var processid = "NA";
     var files = request.getAllFiles();
     for (var name in files) {
         processid = ps.createBPMN(bpmnName, bpmnVersion, files[name].getStream(), user.username);
     }
-    response.sendRedirect('/publisher/assets/process/details/' + processid);
+    //response.sendRedirect('/publisher/assets/process/details/' + processid);
```
Comment (warunalakshitha) [inline modules/extensions/process/publisher/process/apis/upload_bpmn.jag]: remove unwanted comments

Comment (dewmini) [inline modules/extensions/process/publisher/process/apis/upload_bpmn.jag]: Fixed

File: modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs [hunk 2]
```diff
@@ -49,7 +49,7 @@
                                 <input type="hidden" id="docExtension" name="docExtension"/>
                             </div>
 
-                            <div class="span6" style="float:right;width:48%;">
+                            <div class="span6" style="float:right;width:48%;margin-top: 25px;">
```
Comment (dakshika) [inline modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs]: avoid inline styles

Comment (dewmini) [inline modules/extensions/process/publisher/process/themes/default/partials/add_doc.hbs]: This is fixed by https://github.com/wso2/product-pc/commit/e931bb704e9065cf57f08f61619a45a4beecb5c7

**Pull request merged**